.github/workflows/ci.yml

```diff
@@ -6,25 +6,29 @@ on:
       - master
   pull_request:
   schedule:
-    - cron: "50 4 * * *"
+    - cron: "50 6 * * *"
   workflow_dispatch:
 
 env:
-  rust_nightly_toolchain: nightly
+  rust_toolchain: nightly
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
 jobs:
   compile:
     name: Compile
     runs-on: ubuntu-latest
     strategy:
-      fail-fast: false
       matrix:
         target:
           - riscv32imc-esp-espidf
           - xtensa-esp32-espidf
           - xtensa-esp32s2-espidf
           - xtensa-esp32s3-espidf
+        idf-version:
+          - v4.4.6
+          - v5.0
```

Comment:
From a practical standpoint, we might remove v5.0 from the CI runs if we add 5.2, to not make it to slow. I think most people either are still on v4.4.6 or on v5.1.2 currently and that will shift more to 5.2.

Reply:
Fair enough. Removed the pipeline build for v5.0.